File: scratch/merge.js
import fs from 'fs';

const fileContent = fs.readFileSync('src/data/calculatorData.js', 'utf8');

// We'll use a regex to extract the PACKAGES and PACKAGE_PRICES
// Actually, it's easier to just import it since it's ES module, but ts-node/tsx might fail.
// Let's just use a simple regex or execute a dynamic import.

// To execute a dynamic import in Node, we can just run it.

File: scratch/merge.py
import re
import json

with open("src/data/calculatorData.js", "r", encoding="utf-8") as f:
    content = f.read()

packages_match = re.search(r"export const PACKAGES = (\{.*?\});\n\nexport const PACKAGE_PRICES", content, re.DOTALL)
prices_match = re.search(r"export const PACKAGE_PRICES = (\{.*?\});\n\nexport const CREATIVE_SERVICE_PRICES", content, re.DOTALL)

packages_str = packages_match.group(1)
prices_str = prices_match.group(1)

# Hacky way to parse JS objects:
import ast
def js_to_py(js_str):
    # This might be tricky because JS keys are unquoted sometimes.
    pass

# Better approach: since we are in JS environment, let's use node directly.
